src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderLeash.java

```diff
@@ -103,7 +108,7 @@ public boolean hasGlow() {
     public void onInteractEntity(PlayerInteractEntityEvent event) {
         Entity target = event.getRightClicked();
         Player player = event.getPlayer();
-        if (target instanceof Animals && isPassive(target) && player.getItemInHand().getAmount() == 1) {
+        if (target instanceof Animals && isPassive(target) && player.getInventory().getItemInMainHand().getAmount() == 1) {
```

Comment:
This event can be invoked twice, with the off- and the main hand, so you'd need to check for both, by using the appropriate item based on the output of `event.getHand()`

Reply:
Ah yes I missed this as it doesn't trigger twice with leashable animals, only others like turtles. It can't actually be used in the offhand slot though as it does not trigger the event. Should I just only allow the event to proceed if it is from the main hand? Currently if its in your offhand it never triggers the event, but will act as a normal leash for leashable animals.

Reply:
Ah, that explains this inconsistency I experienced with turtles.
In that case yes, simply ignore sll offhand events.

Reply:
What about adding a listener for PlayerLeashEntityEvent to prevent it being used as a normal leash when in the offhand, or shall I just ignore that?

Reply:
Ok so unfortunately it seems that the PlayerLeashEntityEvent also wont trigger from the offhand. I don't really understand how these events are registered and when they will and wont trigger but it seems with the current system this isn't possible. 

This isn't really a big issue, but if you do try to use it on a leashable animal from the offhand with nothing in the main hand. It will leash the animal as usual and on dropping turn back into a normal leash. I could always make an issue for this if required.